README.md

```diff
@@ -126,6 +153,7 @@ __fastify-gql__ supports the following options:
   executed before being jit'ed.
 * `routes`: boolean. Serves the Default: `true`. A graphql endpoint is
   exposed at `/graphql`.
+* `context`: `Function`. Result of function is passed to resolvers as a custom GraphQL context.
```

Comment:
would you mind specifying that the parameters will be `request` and `reply`? Also, we should note that this is called only with the pre-defined routes (`routes: true`).